src/SmartComponents/Inventory/applications/general-information/general-information.scss

```diff
@@ -15,7 +15,7 @@
 }
 
 .pf-c-modal-box .pf-c-table {
-    white-space:nowrap;
+    white-space: nowrap;
```

Comment:
Is this going to cause issues on mobile? I think setting the min-width a larger, percentage based value would be better unless the mocks specifically call for these smaller modals. If that's the case, this should be fine as long as wrapping is not ever needed on smaller screens.